Backport the headphone fixes to the effective ROCKNIX UCM 1.2.13 package.
The WCD headphone mode still lives in the RX macro snippet in this version.
Include RX initialization in the downstream Odin2 and Thor profiles.

Upstream: https://github.com/alsa-project/alsa-ucm-conf/commit/33a372e704aee8f94e3af453d8029c6587faca75
Upstream: https://github.com/alsa-project/alsa-ucm-conf/commit/8e6acab2e5426023c9361c55bf115d80cb940ca5

diff --git a/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneEnableSeq.conf b/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneEnableSeq.conf
--- a/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneEnableSeq.conf
+++ b/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneEnableSeq.conf
@@ -3 +3 @@ EnableSequence
-	cset "name='RX HPH Mode' CLS_H_ULP"
+	cset "name='RX HPH Mode' CLS_H_LOHIFI"
@@ -10,2 +9,0 @@ EnableSequence
-	cset "name='RX_COMP1 Switch' 1"
-	cset "name='RX_COMP2 Switch' 1"
diff --git a/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneDisableSeq.conf b/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneDisableSeq.conf
--- a/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneDisableSeq.conf
+++ b/ucm2/codecs/qcom-lpass/rx-macro/HeadphoneDisableSeq.conf
@@ -8,2 +7,0 @@ DisableSequence
-	cset "name='RX_COMP1 Switch' 0"
-	cset "name='RX_COMP2 Switch' 0"
diff --git a/ucm2/codecs/qcom-lpass/rx-macro/init.conf b/ucm2/codecs/qcom-lpass/rx-macro/init.conf
--- a/ucm2/codecs/qcom-lpass/rx-macro/init.conf
+++ b/ucm2/codecs/qcom-lpass/rx-macro/init.conf
@@ -5,0 +6,2 @@ BootSequence
+	cset "name='RX_COMP1 Switch' 0"
+	cset "name='RX_COMP2 Switch' 0"
diff --git a/ucm2/AYN/Odin2/AYN-Odin2.conf b/ucm2/AYN/Odin2/AYN-Odin2.conf
--- a/ucm2/AYN/Odin2/AYN-Odin2.conf
+++ b/ucm2/AYN/Odin2/AYN-Odin2.conf
@@ -31,0 +32 @@
+Include.rx-init.File "/codecs/qcom-lpass/rx-macro/init.conf"
diff --git a/ucm2/AYN/Thor/AYN-Thor.conf b/ucm2/AYN/Thor/AYN-Thor.conf
--- a/ucm2/AYN/Thor/AYN-Thor.conf
+++ b/ucm2/AYN/Thor/AYN-Thor.conf
@@ -33,0 +34 @@
+Include.rx-init.File "/codecs/qcom-lpass/rx-macro/init.conf"
